Database/Corrections/sodQuestFixes.lua

```diff
@@ -2928,6 +2928,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -402852,
             [questKeys.zoneOrSort] = sortKeys.PRIEST,
         },
+        [90193] = {
+            [questKeys.name] = "Between the Eyes",
+            [questKeys.startedBy] = {nil,{405946}},
```

Comment:
We've got two spawns on this object, which need to be cleaned up.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5577/commits/0e2f6b862bbc5dd39be35e0f57d2cf537691b332